### DISCUSSION TITLE
How to add hydrogen based on the coordinates and the chiral atom? The input file is pdb

### DISCUSSION THREAD

**Discussion by jnutyj** [rdkit/rdkit#8061] [Q&A] [answered]
Hi Rdkit community,
I have a small molecule pdb file that does not contain hydrogen atoms. I want to add the hydrogen to it, and I know we can do that by Chem.AddHs(mol, addCoords=True). But what I want is to also assign the hydrogen based on the chiral atoms. 
To make it clear, here is what I want:
1. keep the original pdb atom coordinates and add hydrogen
2. also add hydrogen based on all possible stereoisomers  
3. save each isomer independently as pdb. 
Thank you!

**Comment by bp-kelley** [comment]
If you are making more than one stereo isomer, how does that not change the original pdb atoms?  Are you dealing with flat time averaged geometries like nitrogens?

**Reply by jnutyj** [reply]
Sorry I should descript it clear. I just want to add the hydrogen based on the chiral center for R or S. 
Actually, the molecule had hydrogens for specific isomer, but after docking, the hydrogen atoms were missing, and I want to add them back.

**Reply by jnutyj** [reply, marked as answer]
I think I found the solution by using the EnmerateStereoisomers and AssignAtomChiralTagsFromStructure.
Thanks a lot anyway!
